.github/actions/backport/action.yml

```diff
@@ -0,0 +1,73 @@
+name: Backport
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+  REPO_LOCATION:
+    required: true
+  TARGET_BRANCH:
+    required: true
+  COMMIT_SHA:
```

Comment:
This name is confusing. It doesn't have to be a SHA, it can be any git ref (tag, branch, commit sha). Maybe `REF_TO_BACKPORT`?

Reply:
0b32856d58db1e20736a01f8a2c2debb87f2b753